src/components/results.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable indent */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] Please do not abuse the disabling of linters, instead, try to find a solution for that issue. 😄
    - Linters checks are part of your project to teach you and help you with the best practices. 🚀